<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-Q37MKND" ignored="0" name="SAAS" tests="50" failures="4" timestamp="21 一月 2020 02:54:21 GMT" time="165.496" errors="0">
  <testcase name="UserInfoQueryData" time="0.453" classname="com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel"/>
  <testcase name="UserInfoQueryData" time="0.22" classname="com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel"/>
  <testcase name="UserInfoQueryData" time="0.156" classname="com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel"/>
  <testcase name="UserInfoQueryData" time="0.145" classname="com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel"/>
  <testcase name="UserInfoQueryData" time="0.159" classname="com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel"/>
  <testcase name="UserInfoQueryData" time="0.135" classname="com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel"/>
  <testcase name="UserInfoQueryData" time="0.117" classname="com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel"/>
  <testcase name="UserInfoRgisterData" time="0.276" classname="com.bitnei.apitest.testcases.saas.UserInforRgisterByExcel"/>
  <testcase name="UserInfoRgisterData" time="0.479" classname="com.bitnei.apitest.testcases.saas.UserInforRgisterByExcel"/>
  <testcase name="UserInfoRgisterData" time="4.49" classname="com.bitnei.apitest.testcases.saas.UserInforRgisterByExcel"/>
  <testcase name="UserInfoRgisterData" time="2.304" classname="com.bitnei.apitest.testcases.saas.UserInforRgisterByExcel"/>
  <testcase name="UserInfoRgisterData" time="0.272" classname="com.bitnei.apitest.testcases.saas.UserInforRgisterByExcel"/>
  <testcase name="UserInfoRgisterData" time="0.124" classname="com.bitnei.apitest.testcases.saas.UserInforRgisterByExcel"/>
  <testcase name="UserInfoRgisterData" time="0.152" classname="com.bitnei.apitest.testcases.saas.UserInforRgisterByExcel"/>
  <testcase name="UserInfoRgisterData" time="0.596" classname="com.bitnei.apitest.testcases.saas.UserInforRgisterByExcel"/>
  <testcase name="UserInfoRgisterData" time="0.284" classname="com.bitnei.apitest.testcases.saas.UserInforRgisterByExcel"/>
  <testcase name="UserInfoRgisterData" time="0.129" classname="com.bitnei.apitest.testcases.saas.UserInforRgisterByExcel"/>
  <testcase name="UserInfoRgisterData" time="0.184" classname="com.bitnei.apitest.testcases.saas.UserInforRgisterByExcel"/>
  <testcase name="UserInfoRgisterData" time="0.324" classname="com.bitnei.apitest.testcases.saas.UserInforRgisterByExcel"/>
  <testcase name="UserInfoRgisterData" time="0.146" classname="com.bitnei.apitest.testcases.saas.UserInforRgisterByExcel"/>
  <testcase name="UserInfoRgisterData" time="0.167" classname="com.bitnei.apitest.testcases.saas.UserInforRgisterByExcel"/>
  <testcase name="UserInfoRgisterData" time="0.105" classname="com.bitnei.apitest.testcases.saas.UserInforRgisterByExcel"/>
  <testcase name="UserInfoRgisterData" time="0.267" classname="com.bitnei.apitest.testcases.saas.UserInforRgisterByExcel"/>
  <testcase name="UserInfoRgisterData" time="0.31" classname="com.bitnei.apitest.testcases.saas.UserInforRgisterByExcel"/>
  <testcase name="UserInfoRgisterData" time="0.103" classname="com.bitnei.apitest.testcases.saas.UserInforRgisterByExcel"/>
  <testcase name="UserInfoQueryData" time="0.429" classname="com.bitnei.apitest.testcases.saas.PostBackSaas"/>
  <testcase name="UserInfoQueryData" time="0.134" classname="com.bitnei.apitest.testcases.saas.PostBackSaas"/>
  <testcase name="UserInfoQueryData" time="0.174" classname="com.bitnei.apitest.testcases.saas.PostBackSaas"/>
  <testcase name="UserInfoQueryData" time="0.138" classname="com.bitnei.apitest.testcases.saas.PostBackSaas"/>
  <testcase name="UserInfoQueryData" time="0.091" classname="com.bitnei.apitest.testcases.saas.PostBackSaas"/>
  <testcase name="UserInfoQueryData" time="0.171" classname="com.bitnei.apitest.testcases.saas.PostBackSaas"/>
  <testcase name="UserInfoQueryData" time="0.144" classname="com.bitnei.apitest.testcases.saas.PostBackSaas"/>
  <testcase name="ApiTestCode" time="0.295" classname="com.bitnei.apitest.testcases.saas.ApiTest"/>
  <testcase name="ApiTestCode" time="0.056" classname="com.bitnei.apitest.testcases.saas.ApiTest"/>
  <testcase name="ApiTestCode" time="0.211" classname="com.bitnei.apitest.testcases.saas.ApiTest"/>
  <testcase name="ApiTestCode" time="0.085" classname="com.bitnei.apitest.testcases.saas.ApiTest"/>
  <testcase name="ApiTestCode" time="0.068" classname="com.bitnei.apitest.testcases.saas.ApiTest"/>
  <testcase name="ApiTestCode" time="0.042" classname="com.bitnei.apitest.testcases.saas.ApiTest">
    <failure type="java.lang.AssertionError" message="expected [{}] but found [{&amp;quot;error&amp;quot;:{&amp;quot;currentValue is &amp;quot;:true,&amp;quot;expectedValue is&amp;quot;:false},&amp;quot;code&amp;quot;:{&amp;quot;currentValue is &amp;quot;:&amp;quot;40000&amp;quot;,&amp;quot;expectedValue is&amp;quot;:false,&amp;quot;expectedValue is &amp;quot;:&amp;quot;10000&amp;quot;}}]">
      <![CDATA[java.lang.AssertionError: expected [{}] but found [{"error":{"currentValue is ":true,"expectedValue is":false},"code":{"currentValue is ":"40000","expectedValue is":false,"expectedValue is ":"10000"}}]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.bitnei.apitest.testcases.saas.ApiTest.ApiTestCode(ApiTest.java:101)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- ApiTestCode -->
  <testcase name="ApiTestCode" time="0.077" classname="com.bitnei.apitest.testcases.saas.ApiTest">
    <failure type="java.lang.AssertionError" message="expected [{}] but found [{&amp;quot;error&amp;quot;:{&amp;quot;currentValue is &amp;quot;:true,&amp;quot;expectedValue is&amp;quot;:false},&amp;quot;code&amp;quot;:{&amp;quot;currentValue is &amp;quot;:&amp;quot;40000&amp;quot;,&amp;quot;expectedValue is&amp;quot;:false,&amp;quot;expectedValue is &amp;quot;:&amp;quot;10000&amp;quot;}}]">
      <![CDATA[java.lang.AssertionError: expected [{}] but found [{"error":{"currentValue is ":true,"expectedValue is":false},"code":{"currentValue is ":"40000","expectedValue is":false,"expectedValue is ":"10000"}}]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.bitnei.apitest.testcases.saas.ApiTest.ApiTestCode(ApiTest.java:101)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- ApiTestCode -->
  <testcase name="ApiTestCode" time="0.073" classname="com.bitnei.apitest.testcases.saas.ApiTest">
    <failure type="java.lang.AssertionError" message="expected [{}] but found [{&amp;quot;error&amp;quot;:{&amp;quot;currentValue is &amp;quot;:true,&amp;quot;expectedValue is&amp;quot;:false},&amp;quot;code&amp;quot;:{&amp;quot;currentValue is &amp;quot;:&amp;quot;40000&amp;quot;,&amp;quot;expectedValue is&amp;quot;:false,&amp;quot;expectedValue is &amp;quot;:&amp;quot;35000&amp;quot;}}]">
      <![CDATA[java.lang.AssertionError: expected [{}] but found [{"error":{"currentValue is ":true,"expectedValue is":false},"code":{"currentValue is ":"40000","expectedValue is":false,"expectedValue is ":"35000"}}]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.bitnei.apitest.testcases.saas.ApiTest.ApiTestCode(ApiTest.java:101)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- ApiTestCode -->
  <testcase name="ApiTestCode" time="0.147" classname="com.bitnei.apitest.testcases.saas.ApiTest"/>
  <testcase name="ApiTestCode" time="0.045" classname="com.bitnei.apitest.testcases.saas.ApiTest">
    <failure type="java.lang.AssertionError" message="expected [{}] but found [{&amp;quot;error&amp;quot;:{&amp;quot;currentValue is &amp;quot;:true,&amp;quot;expectedValue is&amp;quot;:false},&amp;quot;code&amp;quot;:{&amp;quot;currentValue is &amp;quot;:&amp;quot;40000&amp;quot;,&amp;quot;expectedValue is&amp;quot;:false,&amp;quot;expectedValue is &amp;quot;:&amp;quot;10000&amp;quot;}}]">
      <![CDATA[java.lang.AssertionError: expected [{}] but found [{"error":{"currentValue is ":true,"expectedValue is":false},"code":{"currentValue is ":"40000","expectedValue is":false,"expectedValue is ":"10000"}}]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.bitnei.apitest.testcases.saas.ApiTest.ApiTestCode(ApiTest.java:101)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- ApiTestCode -->
  <testcase name="ApiTestCode" time="0.054" classname="com.bitnei.apitest.testcases.saas.ApiTest"/>
  <testcase name="ApiTest" time="0.258" classname="com.bitnei.apitest.testcases.saas.PostApiTestParas"/>
  <testcase name="UserInfoRgisterData" time="4.404" classname="com.bitnei.apitest.testcases.saas.TestGeodeData"/>
  <testcase name="UserInfoRgisterData" time="7.969" classname="com.bitnei.apitest.testcases.saas.TestGeodeData"/>
  <testcase name="UserInfoRgisterData" time="7.059" classname="com.bitnei.apitest.testcases.saas.TestGeodeData"/>
  <testcase name="UserInfoRgisterData" time="15.458" classname="com.bitnei.apitest.testcases.saas.TestGeodeData"/>
  <testcase name="UserInfoRgisterData" time="8.176" classname="com.bitnei.apitest.testcases.saas.TestGeodeData"/>
  <testcase name="UserInfoRgisterData" time="16.077" classname="com.bitnei.apitest.testcases.saas.TestGeodeData"/>
</testsuite> <!-- SAAS -->
